src/pages/HomePage.js

```diff
@@ -0,0 +1,16 @@
+import Sidebar from "../components/Sidebar";
+import Searchbar from "../components/Searchbar";
+import AddNote from "../components/AddNote";
+import ShowNote from "../components/ShowNote";
```

Comment:
In your components folder you can have an index.js file having all these components one by one and then you would be able to write this in a single line one by one:
```
//In the index.js file of components folder
export Sidebar from "../components/Sidebar";
export Searchbar from "../components/Searchbar";
export AddNote from "../components/AddNote";
export ShowNote from "../components/ShowNote";
```
```
//In your homepage.js file
import {Sidebar,Searchbar,AddNote,ShowNote} from '../components'
```